lib/puppet/type/cpan.rb

```diff
@@ -27,6 +27,9 @@
   newparam(:local_lib) do
     desc "Destination directory or `false`"
   end
+  newparam(:umask) do
+    desc 'umask to run cpan with'
```

Comment:
should we add a readme block for explaining this param ?

Reply:
shouldn't we add some tests around this ?

Reply:
I've separately started working on some type/provider unit tests and should hopefully be able to create a PR soon.  That's how I've come across https://github.com/meltwater/puppet-cpan/issues/41 :)